<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-06T03:32:57 IST" name="Suite" finished-at="2023-02-06T03:34:23 IST" duration-ms="86277">
    <groups>
    </groups>
    <test started-at="2023-02-06T03:32:57 IST" name="Login Test" finished-at="2023-02-06T03:33:29 IST" duration-ms="32723">
      <class name="com.orderocks.tests.LoginTest">
        <test-method is-config="true" signature="initialze()[pri:0, instance:com.orderocks.tests.LoginTest@7fd4acee]" started-at="2023-02-06T03:32:57 IST" name="initialze" finished-at="2023-02-06T03:33:28 IST" duration-ms="31336" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialze -->
        <test-method signature="verifyLogin()[pri:0, instance:com.orderocks.tests.LoginTest@7fd4acee]" started-at="2023-02-06T03:33:28 IST" name="verifyLogin" finished-at="2023-02-06T03:33:28 IST" duration-ms="326" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Let's get you started...] but found [Let's get you started]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Let's get you started...] but found [Let's get you started]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.orderocks.tests.LoginTest.verifyLogin(LoginTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orderocks.tests.LoginTest@7fd4acee]" started-at="2023-02-06T03:33:29 IST" name="tearDown" finished-at="2023-02-06T03:33:29 IST" duration-ms="341" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orderocks.tests.LoginTest -->
    </test> <!-- Login Test -->
    <test started-at="2023-02-06T03:33:29 IST" name="Place Order" finished-at="2023-02-06T03:34:23 IST" duration-ms="53471">
      <class name="com.orderocks.tests.OrderTest">
        <test-method is-config="true" signature="initialze()[pri:0, instance:com.orderocks.tests.OrderTest@2cf23c81]" started-at="2023-02-06T03:33:29 IST" name="initialze" finished-at="2023-02-06T03:33:45 IST" duration-ms="15516" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialze -->
        <test-method signature="verifyCreateAnOder()[pri:0, instance:com.orderocks.tests.OrderTest@2cf23c81]" started-at="2023-02-06T03:33:45 IST" name="verifyCreateAnOder" finished-at="2023-02-06T03:34:23 IST" duration-ms="37606" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyCreateAnOder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orderocks.tests.OrderTest@2cf23c81]" started-at="2023-02-06T03:34:23 IST" name="tearDown" finished-at="2023-02-06T03:34:23 IST" duration-ms="344" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orderocks.tests.OrderTest -->
    </test> <!-- Place Order -->
  </suite> <!-- Suite -->
</testng-results>
